### ISSUE TITLE
Request ADB commad to grand services for voxDMR assessibility for lockscreen PTT

### ISSUE THREAD

**Issue by lim3soft** [jcalado/voxdmr-site#85]
i'm facing a lock screen PTT issue on my POC, my poc device android cannot access the accessibility option in the settings. Once it opens accessibility, it will somehow clash.

Since the voxDMR 13.0+1 needs to grant permission in accessibility, I'm thinking of enabling it via ADB. May I know what the service name is for the voxDMR need to be grant? Some resource i found in website ask me to do that, but i need the service name

example : 
1.Run this command to grant the system setting permission:
adb shell pm grant com. voxdmr android. permission.WRITE_SECURE_SETTINGS

2.Run this command to add VoxDMR to your device's enabled accessibility services:
adb shell settings put secure enabled_accessibility_services com.voxdmr/com.voxdmr.YourAccessibilityServiceName

3.Run this final command to activate the accessibility framework:
adb shell settings put secure accessibility_enabled 1

**Comment by jcalado** [comment]
The component name is:

```
com.jcalado.voxdmr/dev.voxdmr.voxdmr.PttAccessibilityService
```

With the device connected:

```
adb shell settings put secure enabled_accessibility_services com.jcalado.voxdmr/dev.voxdmr.voxdmr.PttAccessibilityService
adb shell settings put secure accessibility_enabled 1
```

Note that the first command replaces the whole list, so if you have other accessibility services enabled, check the current value first with `adb shell settings get secure enabled_accessibility_services` and append yours with a `:` separator instead.
